groupie/src/main/java/com/genius/groupie/GroupAdapter.java

```diff
@@ -53,13 +50,12 @@ public int getSpanCount() {
     }
 
     @Override public void onBindViewHolder(RecyclerView.ViewHolder holder, int position) {
-        Item contentItem = getItem(position);
-        contentItem.bind(holder, position, onItemClickListener);
```

Comment:
I should have done this in another commit  : I removed it because it is never going to be called.  
If you override the onBind with payload, the classical one is useless.